srcCpp/customType.h

```diff
@@ -0,0 +1,32 @@
+#ifndef __CUSTOMTYPE_H__
+#define __CUSTOMTYPE_H__
+/*
+ * (c) 2005-2017  Copyright, Real-Time Innovations, Inc. All rights reserved.
+ * Subject to Eclipse Public License v1.0; see LICENSE.md for details.
+ */
+#define Q(x) #x
+#define QUOTE(x) Q(x) // Used to set quotation marks
+
+#include QUOTE(RTI_CUSTOM_TYPE_FILE_NAME_SUPPORT)
+
+#include "perftest.h"
+#include <string>
+#include <new>          // std::bad_alloc
```

Comment:
I don't remember needing this in the code before (and we do use bad_alloc, I believe).